src/server/routes/api.js

```diff
@@ -94,10 +94,13 @@ module.exports = function(app) {
   api.get('/users/:eventAlias', requireAuth, roleAuth(roles.ROLE_ADMIN),
     isOrganiser,
     (req, res) => {
-      return User.find({event: req.event})
+      let query = User.find({event: req.event});
+
+      return query
```

Comment:
Probably doesn't need to be separated into its own variable, but it's not a big deal